checkups/kubevirt-vm-latency/vmlatency/internal/vmi/spec.go

```diff
@@ -59,16 +58,16 @@ func withTerminationGracePeriodSecond(duration int64) Option {
 	}
 }
 
-// WithNodeSelector ensures that the VMI gets scheduled on the specified node.
-func WithNodeSelector(nodeName string) Option {
+// WithLabels adds the given labels.
+func WithLabels(labels map[string]string) Option {
 	return func(vmi *kvcorev1.VirtualMachineInstance) {
-		if nodeName == "" {
-			return
+		if len(vmi.ObjectMeta.Labels) < 1 {
```

Comment:
Will this snippet work?

```suggestion
		if vmi.ObjectMeta.Labels == nil {
```

Reply:
It won't cover the case where `vmi.ObjectMeta.Labels` is initialized but empty, this is why I think its better to use `len` for such checks.

Reply:
Done